src/Php80/Php80.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Php80;
+
+/**
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ *
+ * @internal
+ */
+final class Php80
+{
+    public static function fdiv($dividend, $divisor)
+    {
+        if (PHP_VERSION_ID >= 70000) {
```

Comment:
please use a fully-qualified name for this constant, so that OPCache resolves it at compile-time and drops dead code.